# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../lxrandr/hotfix.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Avoid inf. loop, ...

--- lxrandr-0.1.1/po/Makefile.in.in.vanilla	2010-08-24 09:39:59.059220395 +0000
+++ lxrandr-0.1.1/po/Makefile.in.in	2010-08-24 09:40:10.452293235 +0000
@@ -201,7 +201,7 @@
 	  fi; \
 	done
 
-Makefile POTFILES: stamp-it
+Makefile POTFILES:
 	@if test ! -f $@; then \
 	  rm -f stamp-it; \
 	  $(MAKE) stamp-it; \
